### PR TITLE
chore: add support for k8s version 1.11.8

### PR DESCRIPTION
**Pull request by mboersma**
**Reason for Change**:

See https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.11.md#changelog-since-v1117

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version
- [x] Windows assets zip file uploaded to production blob store

**Notes**:

### CODE REVIEW

File: packer/install-dependencies.sh [hunk 1]
```diff
@@ -95,7 +95,7 @@ for TILLER_VERSION in ${TILLER_VERSIONS}; do
     pullContainerImage "docker" "gcr.io/kubernetes-helm/tiller:v${TILLER_VERSION}"
 done
 
-CLUSTER_AUTOSCALER_VERSIONS="1.13.1 1.12.2 1.3.4 1.3.3 1.2.2 1.1.2"
+CLUSTER_AUTOSCALER_VERSIONS="1.13.1 1.12.2 1.3.7 1.3.3 1.2.2 1.1.2"
```
Comment (jackfrancis) [inline packer/install-dependencies.sh]: Let's feel free to *add* component versions to VHD but retain old versions as well.

Comment (mboersma) [reply packer/install-dependencies.sh]: There are no references to cluster-autoscaler 1.3.4 any longer, so I dropped it from the VHD list.

Comment (mboersma) [reply packer/install-dependencies.sh]: Actually I guess 1.3.3 is also unused now.

Comment (mboersma) [reply packer/install-dependencies.sh]: I removed the removal--we need to audit AKS as well for usage before aging things out of the VHD image.

**Comment by jackfrancis** [comment]
So we actually do want the cluster-autoscaler `v1.3.7` in the VHD, but we don't need the k8s version itself as that modification already landed.

However, as we've already kicked off the VHD, what is the urgency of bumping cluster-autoscaler to 1.3.7 from 1.3.4?

**Comment by mboersma** [comment]
> what is the urgency of bumping cluster-autoscaler to 1.3.7 from 1.3.4?

Here are the changes: https://github.com/kubernetes/autoscaler/compare/cluster-autoscaler-1.3.4...cluster-autoscaler-1.3.7.  This in particular looks desirable: https://github.com/kubernetes/autoscaler/commit/fae2e71d9f05326cc3ae530926f5675a44c5b9b8

File: pkg/api/k8s_versions.go [hunk 2]
```diff
@@ -128,7 +128,7 @@ var k8sComponentVersions = map[string]map[string]string{
 		"aci-connector":                    "virtual-kubelet:latest",
 		ContainerMonitoringAddonName:       "oms:ciprod01092019",
 		AzureCNINetworkMonitoringAddonName: "networkmonitor:v0.0.5",
-		"cluster-autoscaler":               "cluster-autoscaler:v1.3.4",
+		"cluster-autoscaler":               "cluster-autoscaler:v1.3.7",
```
Comment (jackfrancis) [inline pkg/api/k8s_versions.go]: Based on your findings let's definitely keep this change.

Comment (CecileRobertMichon) [reply pkg/api/k8s_versions.go]: why not add 1.3.7 to the VHD script (even if we don't build it right away)?

Comment (mboersma) [reply pkg/api/k8s_versions.go]: Because of cranial flatulence? I've added it to the VHD script now. 😄

**Comment by jackfrancis** [comment]
lgtm pending E2E

**Pull request merged**